### ISSUE TITLE
Subject folders not BIDS compliant sct-testing-large

### ISSUE THREAD

**Issue by NathanMolinier** [neuropoly/data-management#244]
## Description

For some subjects in `sct-testing-large`, the session name was added to the subject name leading to errors in some scripts like [manual-correction](https://github.com/spinalcordtoolbox/manual-correction)

Therefore, the same subject is appearing multiple times as if it was multiple subjects
<img width="407" alt="Capture d’écran 2023-07-04 à 16 42 23" src="https://github.com/neuropoly/data-management/assets/68945192/ee6f008c-6534-406f-b0c8-db9dc4c556d1">

**Comment by valosekj** [comment]
Good catch!

Indeed, `ses` folders are missing and subject folders incorrectly contains `ses`:

```console
$ cd sct-testing-large 
$ tree sub-zurichMPM0*
sub-zurichMPM01_ses-01
└── anat
    ├── sub-zurichMPM01_ses-01_RFSC_MT.json
...
```

The correct directory structure should be:

```console
$ tree sub-zurichMPM0*
sub-zurichMPM01
└── ses-01
    └── anat
        ├── sub-zurichMPM01_ses-01_RFSC_MT.json
...
```

**Comment by mguaypaq** [comment]
I believe the hierarchy is `sub-*/ses-*/anat/...`; with the session folders directly inside the subject folders, and no `anat` in between.

**Comment by NathanMolinier** [comment]
You are right, we have this hierarchy in Canproco I believe

**Comment by valosekj** [comment]
> I believe the hierarchy is `sub-*/ses-*/anat/...`; with the session folders directly inside the subject folders, and no `anat` in between.

You are right! Sorry for this! I overlooked this. I have fixed my comment.
